File: src/main/java/main/constants/MessageButton.java
package main.constants;

public enum MessageButton {
    PAGE_ID("page"),
    FRONT(PAGE_ID + "front"),
    PREVIOUS(PAGE_ID + "previous"),
    NEXT(PAGE_ID + "next"),
    END(PAGE_ID + "end");

    private final String str;

    MessageButton(String str) {
        this.str = str;
    }

    @Override
    public String toString() {
        return str;
    }
}

File: src/main/kotlin/main/constants/MessageButton.kt
package main.constants

enum class MessageButton(private val str: String) {
    PAGE_ID("page"),
    FRONT("${PAGE_ID}front"),
    PREVIOUS("${PAGE_ID}previous"),
    NEXT("${PAGE_ID}next"),
    END("${PAGE_ID}end");

    override fun toString(): String =
        str
}